test/PerformanceTests/SerializationComparisonsTests/Lib/WriterHelpers.cs

```diff
@@ -19,25 +20,25 @@ public static class WriterHelpers
         public static IJsonWriter CreateODataJsonWriter(this Stream stream, Encoding encoding = null)
         {
             return jsonWriterFactory.CreateJsonWriter(
-                new StreamWriter(stream, encoding ?? Encoding.UTF8), false);
+                stream, false, encoding ?? Encoding.UTF8);
         }
 
-        public static IJsonWriterAsync CreateODataJsonWriterAsync(this Stream stream, Encoding encoding = null)
+        public static IJsonWriter CreateODataJsonWriterAsync(this Stream stream, Encoding encoding = null)
         {
-            return jsonWriterFactory.CreateAsynchronousJsonWriter(
-                new StreamWriter(stream, encoding ?? Encoding.UTF8), false);
+            return jsonWriterFactory.CreateJsonWriter(
+                stream, false, encoding ?? Encoding.UTF8);

```

Comment:
Can we use a named parameter here to make it easy for someone to understand what the `false` for example stands for?

Reply:
Update this in all other places. otherwise the pr looks good.

Reply:
Updated.